initialization/initialization_test.go

```diff
@@ -595,9 +600,16 @@ func TestInitialize_MultipleFiles(t *testing.T) {
 			}
 			r.NoError(verifying.VerifyVRFNonce(init.Nonce(), m))
 
-			// TODO(mafa): see above
-			// r.Equal(oneFileNonce, *init.Nonce())
+			if nonce == nil {
+				nonce = init.Nonce()
+			} else {
+				// TODO(mafa): since we are not looking for the absolute lowest nonce, we can't guarantee that the nonce will be the same.
+				// see also https://github.com/spacemeshos/post/issues/90
+				// r.Equal(*nonce, *init.Nonce())
```

Comment:
This assert can now probably be enabled again. Changing the number of files for post should not change the found VRF nonce.

Reply:
It should be the same as before. And it's failing if applied—I guess due to randomness in GPU parallelization.

Reply:
Then this is a task for after genesis; I already created an issue for it: https://github.com/spacemeshos/post/issues/90